spec/factories/api_factory.rb

```diff
@@ -27,6 +24,12 @@
       master_id { SecureRandom.uuid }
       merged { true }
     end
+
+    factory :api_schools_experience_sign_up_with_name do
```

Comment:
Would it be fewer changes to leave first/last name in the main factory and have a trait for `:without_name`?

Reply:
I couldn't set the first_name and last_name to nil in a trait because of the validations, so this was the only workaround I could think of.

Reply:
Ah right I see, makes sense 👍